common/taskana-common-data/src/main/resources/sql/sample-data/task-history-event.sql

```diff
@@ -1,46 +1,46 @@
-INSERT INTO TASK_HISTORY_EVENT VALUES ('THI:000000000000000000000000000000000041', 'BPI:02'           , ''                        , 'TKI:000000000000000000000000000000000000', 'UPDATED' , RELATIVE_DATE(-1), 'USER_1_2', 'DOMAIN_B', 'WBI:100000000000000000000000000000000001', '00'       , 'PASystem', '00'        , 'VNR'   , '11223344', ''                     , ''                          , ''                           , '2old_val' , 'new_val2' , 'custom1', ''       , 'custom3', 'custom4', 'some Details');
-INSERT INTO TASK_HISTORY_EVENT VALUES ('THI:000000000000000000000000000000000042', 'BPI:03'           , 'BPI:01'                  , 'TKI:000000000000000000000000000000000001', 'DELETED' , RELATIVE_DATE(0) , 'USER_2_1', 'DOMAIN_B', 'WBI:100000000000000000000000000000000002', '11'       , ''        , '22'        , ''      , ''        , 'L140101'              , 'TASK'                      , 'DOCTYPE_DEFAULT'            , ''         , ''         , 'custom1', ''       , 'custom3', ''       , 'some Details');
-INSERT INTO TASK_HISTORY_EVENT VALUES ('THI:000000000000000000000000000000000043', 'BPI:04'           , ''                        , 'TKI:000000000000000000000000000000000000', 'CREATED' , RELATIVE_DATE(-1), 'USER_1_2', 'DOMAIN_B', 'WBI:100000000000000000000000000000000001', '00'       , 'PASystem', '00'        , 'VNR'   , '11223344', ''                     , ''                          , ''                           , '2old_val' , 'new_val2' , 'custom1', ''       , 'custom3', 'custom4', 'some Details');
-INSERT INTO TASK_HISTORY_EVENT VALUES ('THI:000000000000000000000000000000000044', 'BPI:03'           , 'BPI:01'                  , 'TKI:000000000000000000000000000000000001', 'DELETED' , RELATIVE_DATE(0) , 'USER_2_1', 'DOMAIN_B', 'WBI:100000000000000000000000000000000002', '11'       , ''        , '22'        , ''      , ''        , 'L140101'              , 'TASK'                      , 'DOCTYPE_DEFAULT'            , ''         , ''         , 'custom1', ''       , 'custom3', ''       , 'some Details');
\ No newline at end of file
+-- TASK_HISTORY_EVENT TABLE           (ID                                        , BUSINESS_PROCESS_ID, PARENT_BUSINESS_PROCESS_ID, TASK_ID                                   , EVENT_TYPE, CREATED          , USER_ID   , DOMAIN    , WORKBASKET_KEY                            , WORKBASKET_NAME, POR_COMPANY, POR_SYSTEM, POR_INSTANCE, POR_TYPE, POR_VALUE , TASK_PRIORITY, TASK_PLANNED         , TASK_DUE             , TASK_OWNER, TASK_CLASSIFICATION_KEY, TASK_CLASSIFICATION_NAME, TASK_CLASSIFICATION_CATEGORY, ATTACHMENT_CLASSIFICATION_KEY, ATTACHMENT_CLASSIFICATION_NAME, OLD_VALUE  , NEW_VALUE  , CUSTOM_1 , CUSTOM_2 , CUSTOM_3 , CUSTOM_4 , DETAILS       );
```

Comment:
see comment for task.sql -> ordering of values to insert != database column order

Reply:
Doesn't apply for test data :)